Add docs for AutoOps on ECK (#4792)
elastic/docs-content: deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md

> # Connect your self-managed cluster to AutoOps

Author: See my comment above for the ECK version bit

"Self-managed cluster" has been approved for this feature in marketing and docs since it came out, always intending to mean ECE, ECK, and self-managed 😭 What should we use instead that will make more sense?

Based on the [Deploy](https://www.elastic.co/docs/deploy-manage/deploy#who-manages-the-infrastructure) page, self-hosted (fully self-managed, ECE, or ECK cluster) could work

> # Connect your self-managed cluster to AutoOps

Reviewer: re: the gating question, I think what is missing now is what ECK people on versions lower than 3.3 should use. the doc only has a clear 3.3+ pathway

### Configure agent

Depending on your selected installation method, you might have to provide some or all of the following information to create the installation command:

* **{{es}} endpoint URL**: Enter the URL for the {{es}} cluster you want to monitor by connecting to AutoOps.
* **Preferred authentication method**: Choose one of the following:
  
  :::::{tab-set}
  :group: api-key-or-basic

  ::::{tab-item} API key
  :sync: api-key

  With this authentication method, you need to create an API key to grant access to your cluster. Complete the following steps:

>   1. Go to {{kib}} in your self-managed cluster.

Author: Updated

4. Return to the wizard and select **I have run the command**.

It might take a few minutes for your cluster details to be validated and the first metrics to be shipped to AutoOps.
:::

:::{tab-item} ECK
:sync: installation-ECK
```{applies_to}
eck: ga 3.3
```
1. Copy the YAML manifest. 
2. (Optional) Paste it into a text editor and change the values of the following variables:
    * Secret name
    * Namespace
    * `resourceSelector` label

> 3. Add the manifest to your ECK environment on the machine where you want to install the agent. 

Reviewer 2: ```suggestion
3. Apply the manifest to your ECK environment.
```

(The k8s scheduler schedules the agent wherever it wants)
